templates/common/bin/common.sh

```diff
@@ -95,9 +95,5 @@ function common_ironic_config {
     crudini --set ${SVC_CFG_MERGED} nova password $IRONICPASSWORD
     crudini --set ${SVC_CFG_MERGED} swift password $IRONICPASSWORD
     crudini --set ${SVC_CFG_MERGED} inspector password $IRONICPASSWORD
-    # TODO: nova password
-    #crudini --set ${SVC_CFG_MERGED} nova password $NOVAPASSWORD
```

Comment:
for context i woudl consier it a security issue if the ironic service had the nova password in its config.

it should not interact with nova as the nova user it should be interacting with nova as the ironic user using ironics password

that is why i remove this todo

Reply:
FWIW, I checked nova's policy, it can be any account with admin rights to post the power status message.

Reply:
yep it can
what im saying is we strong advise against using the nova user to do this.
and i would consider it a security bug in an installer to pass the nova user password to other services.

the other reaseson i don't want to use nova here is if we rotate the nova user password we should not need to restart ironic or neutron

so each service should only use there own user/password to take to the other services to allow rotation of the service keystone password without impacting other services.